index.js

```diff
@@ -36,42 +39,46 @@ var MochaParallelTests = module.exports = function (options) {
         });
 
         runner.on('end', function() {
-            customRunner.start()
+            customRunner.start();
+
             suites.forEach(function (suite) {
                 customRunner.suite(suite);
             });
+
             passes.forEach(function (test) {
                 customRunner.pass(test);
             });
+
             failures.forEach(function (test) {
                 customRunner.fail(test);
             });
-            customRunner.testEnd()
+
+            customRunner.end();
         });
     }
 
     options._.forEach(function (testPath) {
-        var isDirectory = statSync(testPath).isDirectory();
-        var paths = [];
-
-        // if directory get files
-        if (isDirectory) {
-            fs.readdirSync(testPath).forEach(function(file){
-                if (file.substr(-3) === '.js') {
-                    paths.push(testPath + '/' + file)
+        glob(testPath, function (err, files) {
```

Comment:
glob is far more better for files search and it's probably used inside mocha: https://github.com/mochajs/mocha/blob/master/package.json#L282

the reason is simple: you can call mocha test/**/*.js and this is valid.